R/calcFEdemand.R

```diff
@@ -1811,6 +1836,22 @@ calcFEdemand <- function(subtype = "FE") {
                                           getItems(reminditems, "item"))
     description_out <- "useful energy demand in buildings"
   }
+  if (subtype == "FE") {
+    # duplicate SSP2EU scenarios of industry for Navigate scenarios
```

Comment:
I added this duplication of industry SSP2Eu for the new navigate scenarios. Could you take a look, @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q ? Thanks!